src/openvic-simulation/modifier/ModifierManager.cpp

```diff
@@ -1,11 +1,32 @@
 #include "ModifierManager.hpp"
+#include <utility>
+#include "dataloader/NodeTools.hpp"
+#include "modifier/Modifier.hpp"
+#include "modifier/ModifierEffect.hpp"
+
+#define FWD(...) static_cast<decltype(__VA_ARGS__)>(__VA_ARGS__)
 
 using namespace OpenVic;
 using namespace OpenVic::NodeTools;
+using namespace OpenVic::StringUtils;
+
+void ModifierManager::lock_all_modifier_except_base_country_effects() {
+	lock_leader_modifier_effects();
+	lock_unit_terrain_modifier_effects();
+	lock_shared_tech_country_modifier_effects();
+	lock_technology_modifier_effects();
+	lock_base_province_modifier_effects();
+	lock_terrain_modifier_effects();
+}
 
-bool ModifierManager::add_modifier_effect(
-	ModifierEffect const*& effect_cache, std::string_view identifier, bool positive_good, ModifierEffect::format_t format,
-	ModifierEffect::target_t targets, std::string_view localisation_key
+bool ModifierManager::_register_modifier_effect(
+	CaseInsensitiveIdentifierRegistry<ModifierEffect, RegistryStorageInfoDeque>& registry,
+	ModifierEffect::target_t targets,
+	ModifierEffect const*& effect_cache, std::string_view identifier,
```

Comment:
```suggestion
	ModifierEffect const*& effect_cache,
	std::string_view identifier,
```
Make the argument formatting consistent